projects/storefrontapp-e2e-cypress/cypress/helpers/notification.ts

```diff
@@ -218,13 +207,30 @@ export function navigateToPDPInCustomerInterest(productCode: string) {
   });
 }
 
+export function stubForPaginableMyInterests(jsonfile: string, url: string) {
+  cy.server();
+  cy.route('GET', url, `fixture:${jsonfile}`);
+}
+
 export function verifyPagingAndSorting() {
+  stubForPaginableMyInterests(
+    'myinterestpage1.json',
+    `${apiUrl}/rest/v2/electronics-spa/users/current/productinterests?fields=sorts,pagination,results(productInterestEntry,product(code))&sort=name:desc&pageSize=10&lang=en&curr=USD`
+  );
+  stubForPaginableMyInterests(
+    'myinterestpage2.json',
+    `${apiUrl}/rest/v2/electronics-spa/users/current/productinterests?fields=sorts,pagination,results(productInterestEntry,product(code))&sort=name:desc&pageSize=10&currentPage=1&lang=en&curr=USD`
+  );
   navigateToMyInterestsPage();
   cy.get(firstProductCodeSelector).should('contain', firstProductAscending);
   cy.get('.top cx-sorting .ng-select').ngSelect('Name (descending)');
   cy.get(firstProductCodeSelector).should('contain', firstProductDescending);
   cy.get('.cx-product-interests-product-item').should('have.length', 10);
-  cy.get('cx-pagination:first .page-link').should('have.length', 4);
+  cy.get('cx-pagination:first a').should('have.length', 4);
+  cy.get('cx-pagination:first a')
```

Comment:
This fails in mobile e2e test, beause top pagination is hidden on mobile

Reply:
Fixed. Thanks!